docs/under-the-hood/storage.md

```diff
@@ -115,19 +119,56 @@ Minimal size without values: **2-9B**
 
 ### Marvel dataset use case
 
-Taking into account the Marvel dataset, which has 3 labels and 3 property
-indices, the calculation is as follows:
+The Marvel dataset consists of `Hero`, `Comic` and `ComicSeries` labels, which are indexed. There are also three label-property indices - on the `name` property of `Hero` and `Comic` vertices, and on the `title` property of `ComicSeries` vertices. The `ComicSeries` vertices also have the `publishYear` property.
 
-- Nodes x (`Vertex` + `name` property (let's assume each name is on average 10
-chars long, therefor the name on average has 2B+10B) + `SkipListNode` + `next_pointers` +
-`Delta`) = 21723×(112+12+16+16+104)
-- Edges x (`Edge` + `SkipListNode` + `next_pointers` + `Delta`) = 682943×(40+16+16+104)
-- Label Index Entry x (`SkipListNode<LabelIndex::Entry>` + `next_pointers`) = 3×21723×(24+16)
-- Label Property Index Entry x (`SkipListNode<LabelIndex::Entry>` + `name`
-  property + `next_pointers`) = 3×21723×(80+12+16)
+ <img src={require('../data/under-the-hood/marvel-dataset-schema.png').default}/>
 
-In total **~130MB**, but bear in mind the number can vary because objects
-can have higher overhead due to the additional data.
+There are 6487 `Hero` and 12661 `Comic` vertices with the property `name`. That's 19148 vertices in total. To calculate how much storage those vertices and properties occupy, we are going to use the following formula:
+
+$\texttt{NumberOfVertices} \times (\texttt{Vertex} + \texttt{properties} + \texttt{SkipListNode} + \texttt{next\_pointers} + \texttt{Delta}).$
+
+Let's assume the name on average has $2\text{B}+10\text{B} = 12\text{B}$ (each name is on average 10 characters long). When we include the average values, we get:
+
+$19148 \times (112\text{B} + 12\text{B} + 16\text{B} + 16\text{B} + 104\text{B}) = 19148 \times 260\text{B} = 4978480\text{B}.$
```

Comment:
I would use the comma to separate the numbers... it's very hard to read like that... 4,978,480

Reply:
```suggestion
$19,148 \times (112\text{B} + 12\text{B} + 16\text{B} + 16\text{B} + 104\text{B}) = 19,148 \times 260\text{B} = 4,978,480\text{B}.$
```

Reply:
Yep, I responded to that in the comment above. Fine by me